src/pycares.c

```diff
@@ -62,7 +62,7 @@ pycares_methods[] = {
 #ifdef PYCARES_PYTHON3
 static PyModuleDef pycares_module = {
     PyModuleDef_HEAD_INIT,
-    "pycares",              /*m_name*/
+    "_core",              /*m_name*/
```

Comment:
I see tornado speedups module use no prefix in python 3.  https://github.com/tornadoweb/tornado/blob/master/tornado/speedups.c#L37
And I see python3 source code, it seems it calculate module.__name__ automatically if last name of _Py_PackageContext is equal with this setting name
https://github.com/python/cpython/blob/master/Objects/moduleobject.c#L165